tools/run_command.py

```diff
@@ -0,0 +1,12 @@
+import logging
+import subprocess
+
+
+# Echoes and runs an OS command, returning exit status and the captured
+# stdout and stderr as a string array.
+def runCommand(command):
+  logging.info("%s" % command)
+  proc = subprocess.run([command], shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
You can just do `capture_output=True`.

Reply:
@htuch we can't use `capture_output` because envoy_ci internal python runtime version is less than 3.7

Reply:
I was able to use it in https://github.com/envoyproxy/envoy/pull/9258/files#diff-bba1df52a76edae166f0c52c4fed21dbR38, I'm wondering what version issues we're hitting here.

Reply:
@htuch it seems that `api_boost.py` is not defined to run on envoy_ci internal docker runtime because the execution definition isn't written in `do_ci.sh` so test should be passed. But if we add this python script execution definition to that shell script, it may be failed... so unless update docker internal python version, this argument should not work....

Reply:
anyway, add issue https://github.com/envoyproxy/envoy/issues/9339

Reply:
@Shikugawa is this blocking? I think I will have a similar issue with a test I'm working on, so I can take a look. Is there anything else here that we need to sort out?

Reply:
@htuch hmm...can't pass formatting check because of confusing exception...
https://dev.azure.com/cncf/envoy/_build/results?buildId=23684&view=logs&jobId=9fd74226-df5b-5f29-39da-76fc341cc763&j=9fd74226-df5b-5f29-39da-76fc341cc763&t=068b98b1-acf0-52d0-fc6e-f6d1a5b1d5ef

Reply:
so is `capture_output=True` is the reason for python >= 3.7? I think using subprocess.PIPE can workaround it.

@Shikugawa the exception seems from yapf so it's failing on Python formatter somewhere, were you able to run `ci/run_envoy_docker.sh 'ci/check_and_fix_format.sh'` locally?